src/tool/subcommands/index_cmd.rs

```diff
@@ -32,9 +33,12 @@ pub enum IndexCommands {
         /// The starting tipset epoch for back-filling (inclusive), defaults to chain head
         #[arg(long)]
         from: Option<ChainEpoch>,
-        /// The ending tipset epoch for back-filling (inclusive)
+        /// Ending tipset epoch for back-filling (inclusive)
         #[arg(long)]
-        to: ChainEpoch,
+        to: Option<ChainEpoch>,
+        /// Number of tipsets for back-filling
+        #[arg(long, conflicts_with = "to")]
+        n_tipsets: Option<usize>,
```

Comment:
I think we can remove both the flags and create a new flag `limit: RangeSpec`

 @elmattic what do you think ?

Reply:
Yeah I think so, this is one of the exampe: https://github.com/ChainSafe/forest/blob/4bd105630a535607bc04e5b318b441c18cae3b10/src/cli/subcommands/chain_cmd.rs#L46


But I guess you need to create parser kind of thing:

```
impl std::str::FromStr for RangeSpec {
    type Err = String;

    fn from_str(s: &str) -> Result<Self, Self::Err> {
        if let Some(epoch_str) = s.strip_prefix("e:") {
            epoch_str
                .parse::<ChainEpoch>()
                .map(RangeSpec::To)
                .map_err(|_| format!("Invalid epoch number: {}", epoch_str))
        } else {
            s.parse::<usize>()
                .map(RangeSpec::NumTipsets)
                .map_err(|_| format!("Invalid tipset count: {}", s))
        }
    }
}
```

Reply:
I don't think it's a good idea regarding UX.